enterprise/backend/test/metabase_enterprise/audit_db_test.clj

```diff
@@ -73,3 +74,21 @@
   (is (= #{"plugins/instance_analytics/collections"
            "plugins/instance_analytics/databases"}
          (set (map str (fs/list-dir "plugins/instance_analytics"))))))
+
+(defn- get-audit-db-triggers []
+  (t2/query
```

Comment:
don't use this level of trigger access. check out 
```clojure
task=> (->> (metabase.task/scheduler-info)
            :jobs
            (filter (comp #{"metabase.task.sync-and-analyze.job"} :key))
            first
            :triggers
            (map (comp clojurewerkz.quartzite.conversion/from-job-data :data)))
({"db-id" 1}
 {"db-id" 121}
 {"db-id" 122}
 {"db-id" 13371337}
 {"db-id" 785})
```

Reply:
Make sense: I am having it use metabase.task/scheduler-info, instead of the qrtz table directly. `lojurewerkz.quartzite.conversion/from-job-data` didn't actually change anything, since that output is just the value at the `:data` key.